core/src/components/item/test/inputs/item.e2e.ts

```diff
@@ -0,0 +1,108 @@
+import { expect } from '@playwright/test';
+import type { E2EPage } from '@utils/test/playwright';
+import { test } from '@utils/test/playwright';
+
+test.describe('item: inputs', () => {
+  test('should not have visual regressions', async ({ page }) => {
```

Comment:
At least one of the datetimes is missing entirely here:
- `core/src/components/item/test/inputs/item.e2e.ts-snapshots/item-inputs-ios-rtl-Mobile-Firefox-linux.png`
- `core/src/components/item/test/inputs/item.e2e.ts-snapshots/item-inputs-md-ltr-Mobile-Safari-linux.png`
- `core/src/components/item/test/inputs/item.e2e.ts-snapshots/item-inputs-md-rtl-Mobile-Safari-linux.png`

You may need to do `await page.waitForSelector('.datetime-ready')`?